src/common/lru.c

```diff
@@ -215,15 +218,6 @@ daos_lru_ref_hold(struct daos_lru_cache *lcache, void *key,
 	if (link != NULL) {
 		llink = link2llink(link);
 		D_ASSERT(llink->ll_evicted == 0);
-		if (llink->ll_evicting) {
```

Comment:
If we remove this check, looks like it will be possible that wakeup() might be still called several times...

Reply:
Item will be evicted so no one will hold after that, it won't cause issue.